drivers/dma/dma_mcux_edma.c

```diff
@@ -30,8 +30,14 @@ LOG_MODULE_REGISTER(dma_mcux_edma, CONFIG_DMA_LOG_LEVEL);
 #define HAS_CHANNEL_GAP(n)		DT_INST_NODE_HAS_PROP(n, channel_gap) ||
 #define DMA_MCUX_HAS_CHANNEL_GAP	(DT_INST_FOREACH_STATUS_OKAY(HAS_CHANNEL_GAP) 0)
 
+#if defined(CONFIG_DMA_MCUX_EDMA_V5)
```

Comment:
The same SOC can have different version of EDMA instances. Say i.MX93 has EDMA3 and EDMA4, some may even have EDMA3,4,5 simutaniously. Such code may cause error.